spec/generators/rails-template.rb

```diff
@@ -128,7 +134,7 @@ class Wizard < ApplicationRecord
       has_one :wand
       has_many :spell_books
 
-      scope :recent, -> { where('created_at > ?', 1.month.ago) }
+      scope :recent, -> { where('created_at > ?', 1.month.ago) }#{attachments}
```

Comment:
Is this change intentional? Looks like `#{attachments}` should be removed
```suggestion
      scope :recent, -> { where('created_at > ?', 1.month.ago) }
```

Reply:
This is indeed correct, it’s the hack I used to generate the model with active storage methods in 5.2 and 6.0. You can’t put it on a new line or it causes blank lines in older versions

Reply:
Can we put it in a new line? I'd prefer having an extra line than having attachments here on this line. People will be confused by it later for sure.